src/Symfony/Component/Security/Core/Authorization/Voter/VoterInterface.php

```diff
@@ -31,10 +31,10 @@
      * ACCESS_GRANTED, ACCESS_DENIED, or ACCESS_ABSTAIN.
      *
      * @param TokenInterface $token      A TokenInterface instance
-     * @param object|null    $object     The object to secure
+     * @param mixed          $subject    The subject to secure
```

Comment:
Yes, that BC break is intended so that the interface actually allows to be used like the new `Voter` class introduced in 2.8.